br/README.md

```diff
@@ -1,131 +1,152 @@
 
-When BR is built successfully, you can find binary in the `bin` directory.
+When TiKV-BR is built successfully, you can find binary in the `bin` directory.
 
-## Quick start(docker-compose)
+## Quick Start
 
-```sh
-# Start TiDB cluster
-docker-compose -f docker-compose.yaml rm -s -v && \
-docker-compose -f docker-compose.yaml build && \
-docker-compose -f docker-compose.yaml up --remove-orphans
+```bash
+# Using tiup to start a TiKV cluster and record the PD_ADDR
+tiup playground --db 0 --pd 1 --kv 3 --monitor
+
+# Using go-ycsb to generate test data.
+git clone git@github.com:pingcap/go-ycsb.git
+cd go-ycsb; make
+./bin/go-ycsb load tikv -P workloads/workloada -p tikv.pd="${PD_ADDR}:2379" \
+    -p tikv.type="raw" -p recordcount=100000 -p operationcount=100000 --threads 100
+
+# Backup ycsb test data.
+bin/tikv-br backup raw \
+	-s s3://backup-data/2022-09-16/_test/ \
+	--pd ${PD_ADDR}:2379 \
+	--log-file backup_test.log \
 
-# Attach to control container to run BR
-docker exec -it br_control_1 bash
+# Restore from the backup.
+bin/tikv-br restore raw \
+	-s s3://backup-data/2022-09-16/_test/ \
+	--pd ${PD_ADDR}:2379 \
+	--log-file restore_test.log
+```
 
-# Load testing data to TiDB
-go-ycsb load mysql -p workload=core \
-    -p mysql.host=tidb -p mysql.port=4000 -p mysql.user=root \
-    -p recordcount=100000 -p threadcount=100
+## Deploy 
 
-# How many rows do we get? 100000 rows.
-mysql -uroot -htidb -P4000 -E -e "SELECT COUNT(*) FROM test.usertable"
+### Recommended Deployment Configuration
+- In production environments, deploy `TiKV-BR` on a node with at least 8 cores CPU and 16 GB memory. Select an appropriate OS version by following [Linux OS version requirements](https://docs.pingcap.com/tidb/dev/hardware-and-software-requirements#linux-os-version-requirements).
 
-# Build BR and backup!
-make build && \
-bin/br backup full --pd pd0:2379 --storage "local:///data/backup/full" \
-    --log-file "/logs/br_backup.log"
+- Save backup data to Amazon S3 or one mounted network disk on all `TiKV-BR` and `TiKV` nodes.
 
-# Let's drop database.
-mysql -uroot -htidb -P4000 -E -e "DROP DATABASE test; SHOW DATABASES;"
+- Allocate sufficient resources for backup and restoration.
 
-# Restore!
-bin/br restore full --pd pd0:2379 --storage "local:///data/backup/full" \
-    --log-file "/logs/br_restore.log"
+TiKV-BR, TiKV nodes, and the backup storage system should provide network bandwidth that is greater than the backup speed. If the target cluster is particularly large, the threshold of backup and restoration speed is limited by the bandwidth of the backup network.
+The backup storage system should also provide sufficient write/read performance (IOPS). Otherwise, the IOPS might become a performance bottleneck during backup or restoration.
+TiKV nodes need to have at least two additional CPU cores and high performance disks for backups. Otherwise, the backup might have an impact on the services running on the cluster.
```

Comment:
Suggest: "TiKV nodes need to have at least two additional spared CPU cores and disk bandwidth (related to `ratelimit` parameter) for backups.

And it would be better to have a section talking about "impact of backup & restore" (maybe in another PR). I think this is another topic that user would concern much.

We can refer to https://docs.pingcap.com/tidb/stable/backup-and-restore-faq#how-much-impact-does-a-backup-operation-have-on-the-cluster.

Reply:
Ok, I will add one more section to talk about the performance impact in another PR.